docs/tutorials/advanced-guides/how-to-login-to-toolkit-using-ledger.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'How to log in to the Ultra Toolkit using Ledger'
+order: -99997
+oultine: [0, 5]
+---
+
+# Tutorial - Log in to the Ultra Toolkit using Ledger
+
+Ultra Toolkit is a decentralized application facilitating seamless interaction with the Ultra blockchain and its smart contracts. Designed for developers, it provides essential functionalities such as interacting with Ultra smart contracts, searching for Uniqs, Uniq Factories and Users.
+
+This tutorial will cover the simplest process of logging in into the Ultra Toolkit using a [Ledger Device](https://www.ledger.com).
+Ledger devices are hardware crypto wallets that store your private keys offline.
+
+## Prerequisites
+
+-   You must have a [Ledger Device](https://www.ledger.com). For the sake of this tutorial, we are going to use a Ledger Nano X.
+-   You must have already an account created via official [Ultra client](https://ultra.io/) or Ultra Wallet extension. Refer to the [How to create an Ultra Pro Wallet using Ultra Wallet Extension](../guides/how-to-create-ultra-pro-wallet.md) tutorial to create an account.
+-   Your account must have sufficient UOS tokens for a new account creation transactions fees. Current Ultra Pro Wallet creation price is <OracleConversion :amount="2.0" scope="......2nf5.o4" :param="70000" />.
```

Comment:
If you would just give a link to the dedicated Ultra Pro wallet tutorial you could remove this part